tools/bucket-stats/main.go

```diff
@@ -0,0 +1,120 @@
+/**
+ * Explore DB contents
+ *
+ * Given a beacon-chain DB, This tool provides many option to
+ * inspect and explore it.
+ * i.e. For every bucket, print the number of rows, bucket size, min/average/max size of values
+ */
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	log "github.com/sirupsen/logrus"
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
+	datadir = flag.String("datadir", "", "Path to data directory.")
+)
+
+const (
+	DatabaseFileName = "beaconchain.db"
```

Comment:
Could we specify this as a flag? We have a slasher.db we also might want to inspect.

Reply:
cool. will do it.